### ISSUE TITLE
[lit-next] Exporting a `directives.js` just like decorators.js for all directives

### ISSUE THREAD

**Issue by motss** [lit/lit#1736]
## Description
- Who wants the functionality
   - Everyone.
- What it is they want
   - For consistency's sake, there is already a `decorators.js` exported that includes all decorators from the `/decorators` folders. There should be a `directives.js` that follows the same so that we can import `lit/directives` just like how we do with `lit/decorators`.
      ```ts
      import { cache } from 'lit/directives';
      // or import { cache } from 'lit/directives.js';
      ```
- Why they want it
   - For consistency's sake and the ease of importing >= 1 directive using just 1 import statement:
      ```ts
      import { cache, repeat } from 'lit/directives';
      // or import { cache, repeat } from 'lit/directives.js';
      ```
- Functional description of task/subtask
   1. Exporting all directives from the `/directives` folder just like `/decorators` in a `directives.js` at the root level.
   1. Include the following export map for `directives.js`:
      ```json
      ...
      "exports": {
         ...
          "./directives.js": {
             "default": "./directives.js"
          },
         ...
      }
      ```

## Acceptance criteria
What the card must do in order to accept it as complete. Acceptance Criteria must be concrete or measurable.

1. Multiple directives can be imported using just 1 import statement with node.js `exports` map
   ```ts
   import { cache, repeat } from 'lit/directives';
   ```
1. Multiple directives can be imported using just 1 import statement with node.js `exports` map and with `.js` extension
   ```ts
   import { cache, repeat } from 'lit/directives.js';
   ```
1. Multiple directives can be imported individually (existing behavior and it should preserve)
   ```ts
   import { cache } from 'lit/directives/cache';
   import { repeat } from 'lit/directives/repeat';

   // or import { cache } from 'lit/directives/cache.js';
   // or import { repeat } from 'lit/directives/repeat.js';
   
   ```

**Comment by petergaal91** [comment]
Related [web-test-runner](https://github.com/modernweb-dev/web/issues/1355) issue

```
 🚧 Browser logs:
      TypeError: Failed to fetch dynamically imported module: action-panel.component.spec.ts

 ❌ Could not import your test module. Check the browser logs or open the browser in debug mode for more information.

Error while transforming src\elements\action-panel\action-panel.component.ts: Could not resolve import "lit/decorators".
  11 | };
  12 | import { LitElement, html } from "lit";
> 13 | import { property } from "lit/decorators";
     |                        ^

```

I want to use the migration guide suggested imports wit web test runner, but this not working just when I add a .js suffix:
```
import { property } from 'lit/decorators.js';
```

Could you guys implement an export map to resolve this issue?

**Comment by justinfagnani** [comment]
I commented on the PR, but I really don't think we want to do this:

> I don't think we want to do this. The total collection of directives is much more heavyweight than decorators and there are some very rarely used directives that we shouldn't encourage developers to load by default.

We already separated decorators out from the main export so JS devs don't have to load them. It would honestly be even better for developers to individually import each decorator, but given how often at least `customElement`, `property` and usually `state`, `query`, and `queryAll`, are needed, and how small some of the rarely used directives are, it's ok to bundle them.

Directives are different, some are larger, some are smaller, and there's little correlation between using one and using the rest. A developer just using `styleMap()` should not be loading `repeat()` and `asyncAppend()` for instance.
